templates/educationServices.html

```diff
@@ -441,10 +441,10 @@ <h1 class="h2 mb-4 font-weight-bold text-primary text-center">Educational Materi
                   <td data-column="Support">N/A</td>
                 </tr>
                 <tr>
-                  <td data-column="Name"><a href="https://kb.beaconcha.in/">Beaconcha.in KB</a></td>
+                  <td data-column="Name"><a href="https://kb.agorascan.io/">www.agorascan.io KB</a></td>
                   <td data-column="Author"><a href="https://twitter.com/Butta_eth">Butta</a></td>
                   <td data-column="Type">General Knowledge</td>
-                  <td data-column="Association"><a href="https://twitter.com/beaconcha_in">beaconcha.in </a></td>
+                  <td data-column="Association"><a href="https://t.me/bosagora_eng">www.agorascan.io </a></td>
```

Comment:
```suggestion
                   <td data-column="Association"><a href="https://twitter.com/bosagora1">www.agorascan.io </a></td>
```
I'm sorry. We had a Twitter site